MekHQ/src/mekhq/gui/view/AtBScenarioViewPanel.java

```diff
@@ -174,7 +178,7 @@ private void initComponents() {
     }
 
     private void fillStats() {
-        ResourceBundle resourceMap = ResourceBundle.getBundle("mekhq.resources.AtBScenarioViewPanel",
+        ResourceBundle resourceMap = ResourceBundle.getBundle("mekhq.resources.ScenarioViewPanel",
```

Comment:
The AtBScenarioViewPanel resource bundle is entirely a subset of the ScenarioViewPanel resource bundle. It seemed redundant, so I just used the latter.